### PR TITLE
Add short description of some plugins

### PR DESCRIPTION
**Pull request by AbdulkadirA**
That's a starter for improving the documentation. Needs some more work.

### CODE REVIEW

File: docs/contents/userguide.rst [hunk 1]
```diff
@@ -101,25 +101,41 @@ I/O
...
+If a suitable ``neuroHarmonize`` model is loaded (plugin ``Harmonization``),
+the harmonized muse volumes (i.e. ``H_MUSE_Volume_*``) will also show the
+age-conditional normative range for an average intracranial volume.
+
+.. figure:: images/ScreenshotAgeTrend.png
+
 
 MUSE Harmonization
 ==================
 TODO: Add screenshot and description
```
Comment (ashishsingh18) [inline docs/contents/userguide.rst:122]: @AbdulkadirA I guess you missed this one.

**Review by ashishsingh18 — CHANGES_REQUESTED** [review]
looks fine except for one location which still has TODO

File: docs/contents/userguide.rst (hunk 1 above)
Comment (AbdulkadirA) [reply docs/contents/userguide.rst:122]: @melhemr Could you merge this PR before you add yours?

**Review by melhemr — APPROVED** [review]

**Comment by melhemr** [comment]
@AbdulkadirA seems that @ashishsingh18  has to approve this before I can merge.

**Review by ashishsingh18 — APPROVED** [review]
looks fine

**Pull request merged**